src/frontend/Komponenter/Personoversikt/InntektForPerson.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from 'react';
+import { IFagsakPerson } from '../../App/typer/fagsak';
+import { AxiosRequestConfig } from 'axios';
+import { useDataHenter } from '../../App/hooks/felles/useDataHenter';
+import DataViewer from '../../Felles/DataViewer/DataViewer';
+import { PensjonsgivendeInntekt } from '../../App/typer/personinntekt';
+import { Table } from '@navikt/ds-react';
+import styled from 'styled-components';
+import { Ressurs } from '../../App/typer/ressurs';
+
+export const InntektForPerson: React.FC<{
+    fagsakPerson: IFagsakPerson;
+}> = ({ fagsakPerson }) => {
+    const inntektConfig: AxiosRequestConfig = useMemo(
+        () => ({
+            method: 'GET',
+            url: `/familie-ef-sak/api/naeringsinntekt/fagsak-person/${fagsakPerson.id}`,
+        }),
+        [fagsakPerson]
+    );
```

Comment:
Når skal man bruke `useMemo` vs. useCallback?

Reply:
https://procoders.tech/blog/difference-between-usememo-and-usecallback/#:~:text=UseCallback%20is%20used%20to%20optimize,solutions%20are%20not%20so%20different.